whatis277/harvest
# bean: Add paymentmethods controller

Moves the paymentmethod create and delete handlers to the paymentmethods controller

Testing instructions:
1. `dc up --build bean`
2. Ensure there's no errors
3. Visit http://localhost:8080/home
4. Create a new card
5. Ensure it creates successfully
6. Delete the new card
7. Ensure it deletes successfully
